src/base_processor.py

```
@@ -465,79 +465,484 @@ def prepare_data(self, input_file: Path, nrows: int = None) -> List[dict]:
+                    )
+                    print(
+                        f"✅ CSV carregado com sucesso: encoding={config['encoding']}, sep='{config['sep']}'"
+                    )
+                    print(f"   Colunas detectadas: {list(df.columns)}")
+                    return df
+                except Exception as e:
+                    print(f"   Tentativa {i+1} falhou ({config}): {str(e)}")
+                    if i == len(csv_configs) - 1:
+                        raise Exception(
+                            f"Não foi possível carregar o arquivo CSV após {len(csv_configs)} tentativas"
+                        )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)